pkg/networkserver/downlink.go

```diff
@@ -1241,6 +1253,7 @@ func (ns *NetworkServer) attemptClassADataDownlink(ctx context.Context, dev *ttn
 			Payload:              genDown.Payload,
 			RawPayload:           genDown.RawPayload,
 			DownlinkEvents:       genState.EventBuilders,
+			SessionKeyID:         dev.GetSession().GetSessionKeyID(),
```

Comment:
This is correct, but I would have loved if we made the session key ID part of `genDown` (So add it to `generatedDownlink`).